tests/mocks/pocketjs.ts

```diff
@@ -0,0 +1,302 @@
+import { Mock, It } from 'moq.ts'
+import { DEFAULT_POCKET_CONFIG } from '../../src/config/pocket-config'
+import {
+  Pocket,
+  Configuration,
+  HttpRpcProvider,
+  Session,
+  SessionManager,
+  SessionHeader,
+  Node,
+  StakingStatus,
+  RelayResponse,
+  RpcError,
+  RelayRequest,
+  RelayPayload,
+  HTTPMethod,
+  RelayProofResponse,
+  PocketAAT,
+  RelayMeta,
+  RelayProof,
+  RequestHash,
+} from '@pokt-network/pocket-js'
+
+const MINIMUM_NODE_STAKE = 15150
+
+export const DEFAULT_NODES = [
+  new Node(
+    '4decdda1c176daf9d70f482e1d7ac476eb57b7ae',
+    '3babba8b6a4a3d94e6a3d01d9e1ac15f8d5331f605b06421e5be1720d5a56867',
```

Comment:
just for curiosity's sake, from where did we get these addresses/pubkeys?

Reply:
all of the info is randomly generated, the only one pseudo-random was the transaction sign from the [Relay Proof Response](https://github.com/pokt-foundation/portal-api/pull/197/files#diff-c501755be6dffc4497c51b96bdbbb997595ef498cbb6a1d351c0c96248182e3eR267-R275). Which was the format was taken from a random transaction to satisfy the proof, but due to having the other info random it can't be traced